ios-base/Manager/SessionManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SessionDataManager.swift
+//  ios-base
+//
+//  Created by Juan Pablo Mazza on 11/8/16.
```

Comment:
I think this should be removed, what do you think?

Reply:
Would you change all files creators to Rootstrap? or just remove the ones that are no longer part of the organization?

Reply:
After discussing with @matiasmansilla1989, we don't see anything valuable to have this, you can create another PR removing this line in all the files

Reply:
@MaicolBen these lines are generated automatically by Xcode when creating new files.

Reply:
Let it as it is, thanks